mula/scheduler/config/settings.py

```diff
@@ -36,10 +48,12 @@ class Settings(BaseSettings):
     span_export_grpc_endpoint: Optional[str] = Field(None, env="SPAN_EXPORT_GRPC_ENDPOINT")
```

Comment:
Also: maybe change `SCHEDULER_RABBITMQ_DSN` to be in line with `QUEUE_URI` which is used in the other modules??

Reply:
It would make sense as a more generic name and if we later wish to replace the queue backend (currently `rabbitmq`)